packages/new-polymath-issuer/package.json

```diff
@@ -85,6 +85,7 @@
     "@types/react-dom": "^16.0.11",
     "@types/react-redux": "^6.0.10",
     "@types/redux-mock-store": "^1.0.0",
+    "@types/styled-components": "^4.1.8",
```

Comment:
I think we should have all the types in the main `package.json` so we avoid version clashes etc.

Reply:
The problem is that if an npm package is downloaded from npm (imagine `polymath-ui`) it will not contain all its dependencies specified in the package.json and therefore it will fail.

This means that we must list *all* dependencies that each individual package requires.

Reply:
If we support npm for individual packages, then it means they need to be transpiled and not expose raw TS. In that case if they are transpiled, the types dependencies are not needed for the package to be used.

Reply:
@Grsmto not the type dependencies but what about the content of node_modules?

And yes, they will be transpiled on publish as they should be, this doesn't mean that we are bundling in there its dependencies I believe though

Reply:
Oh wait @Grsmto I didn't realize we were talking about type dependencies specifically, but @types depend on the dependency every individual package has though right? So if they match then they will be hoisted and shared, but otherwise they are package-specific... I'm not fully convinced I'm right though, adding a Tech-debt task in Asana to make a decision. I'd tackle this separately from this PR though, just move all type deps everywhere to the root package.json